tests/integration/backup_tests/helpers.py

```diff
@@ -132,3 +108,12 @@ async def insert_unwanted_data(ops_test: OpsTest) -> None:
     test_collection.insert_one({"unwanted_data": "bad data 2"})
```

Comment:
We should delete this helper. It seems to be a broken precursor (reference to missing functions etc..) of the one you added below